sandbox/libs/authn/src/main/java/org/opensearch/authn/jwt/JwtVendor.java

```diff
@@ -42,6 +42,16 @@ static JsonWebKey getDefaultJsonWebKey() {
         return jwk;
     }
 
+    static JsonWebKey getSmallerWebKey() {
```

Comment:
If some of the functions here are only used for test purposes, can they be moved into a utility class in `src/test/java` somewhere?

Reply:
What does smaller mean here?

Reply:
So basically you have to use the same encryption type but use the 'smaller' bits version i.e. use 256 bits instead of 512. Swapping to a different algorithm entirely was proving to be more of a pain but I did not want to rename it getHSKey etc because it would not be clear from the name alone. 

I am swapping this to getHS256Key instead though because I realize it is not as clear to everyone else.